### ISSUE TITLE
Error when generating POI - work around provided.

### ISSUE THREAD

**Issue by Soenhay** [overviewer/Minecraft-Overviewer#1326]
Hi,

This is my first time posting a bug for something on GitHub. I just created my GitHub account! I hope it is helpful.

Thanks,

Soenhay

**OS**: win7 Home

**Error message from command window:**

2016-08-27 15:09:13 E An error has occurred. This may be a bug. Please let us kn
ow!
See http://docs.overviewer.org/en/latest/index.html#help

This is the error that occurred:
Traceback (most recent call last):
  File "overviewer.py", line 608, in <module>
  File "overviewer.py", line 130, in main
  File "overviewer_core\aux_files\genPOI.pyc", line 540, in main
  File "overviewer_core\aux_files\genPOI.pyc", line 244, in save_cache
  File "overviewer_core\files.pyc", line 164, in __exit__
WindowsError: [Error 183] Cannot create a file when that file already exists

**Here are the contents of my batch file**:
overviewer.exe --config=config1.py --genpoi
pause

**Here is the workaround**:
-Manually delete the "uuidcache.dat" file in the output directory.
OR a better solution...
-Add this to the beginning of your batch file where outputdir is the same as in the config1.py file:
#del "OUTPUTDIR\uuidcache.dat"

**Comment by LeadMagnet** [comment]
This is the same as issue #1325 and #1271 (original).

I did the same as you to correct the issue, expect I deleted the entire content of the output render  folder (just to be safe).

My server is running 2012R2, IIS is doing some weird things with this file (uuidcache.dat). In the end I changed the global permissions for the folder to allow any updates from SYSTEM, NETWORK SERVICE and the 2 IIS user accounts. Has worked so far, although CP says it isn't a perms issue.

Either way, this problem has already been reported and in work.
